kclvm/sema/src/resolver/type_erasure.rs

```diff
@@ -0,0 +1,30 @@
+use kclvm_ast::ast;
+use kclvm_ast::walker::MutSelfMutWalker;
+
+#[derive(Default)]
+struct TypeErasureTransformer {}
+
+impl<'ctx> MutSelfMutWalker<'ctx> for TypeErasureTransformer {
+    fn walk_schema_attr(&mut self, schema_attr: &'ctx mut ast::SchemaAttr) {
+        if let kclvm_ast::ast::Type::Function(_) = schema_attr.ty.as_ref().node {
+            schema_attr.type_str.node = "function".to_string();
```

Comment:
Use the constant for the string literal "function".

Reply:
fixed.